src/docs/ftw-availability/availability-management/index.md

```diff
@@ -0,0 +1,138 @@
+
+Providers can also block availability by using seats. Setting 0 seats
+for the duration of the availability exception sets the listing as
+unavailable for that period. If the period has existing bookings, the
+availability exception will not affect them, so it only blocks future
+bookings.
+
+Default availability, availability exceptions, and bookings are only
+visible to the provider of the listing. When other users view the
+listing’s availability on the listing page, they only see whether the
+listing is available, but they cannot see whether the availability
+results from the availability plan, exceptions, or existing bookings.
+
+## Listing search and timezone coverage
+
+The operator can allow users to search for listings based on their
+availability on certain dates. The date search cannot be restricted to
+specific time zones. This means that to cover listings from all possible
+time zones, the template prolongs the start and end times before sending
+them to the Sharetribe API. See the relevant code
+[in SearchPage.duck.js](https://github.com/sharetribe/web-template/blob/main/src/containers/SearchPage/SearchPage.duck.js#L185-L186).
+
+To be exact, the start time is moved 14 hours earlier, and the end time
+is moved 12 hours later. This means that the availability search results
+may include listings that are available before or after the time frame
+that the user enters.
+
+![Time range](time-range.png)
+
+## Filtering timeslot with intervals
+
+When querying timeslots for a listing using the booking unit "fixed" or
+"hourly", the template uses
+[interval-based filtering](/references/availability). This approach
+allows retrieving significantly less data from the API than using
+regular timeslot queries (the API is restricted to 500 timeslots per
+page). The query used in the template matches the first available
+timeslot within the defined interval, and returns that. See the full
+query in
+[ListingPage.duck.js](https://github.com/sharetribe/web-template/blob/main/src/containers/ListingPage/ListingPage.duck.js#L495)
+This information is used to display the availability of a listing in the
+monthly calendar view when booking:
+
+![Time range](april-availability.png)
+
+For listings using the fixed booking slot unit type, the template
+extends the end time of the timeslot query. If the duration of the fixed
+booking slot is e.g. 2 hours, the end time of the query is extended by 2
+hours. This allows for the timeslot query to return availability
+correctly for the last moments of the month. For example, to display
+availability on the last day of the month at 23:55 for a 2 hour fixed
+booking slot, the template needs to check if there is availability 2
+hours onwards from 23:55, therefore extending the query into the next
+month. See the
+[relevant code in ListingPage.duck.js](https://github.com/sharetribe/web-template/blob/main/src/containers/ListingPage/ListingPage.duck.js#L488-L493).
```

Comment:
I think this misses 1 paragraph that states how the date picker works with 'hour' and 'fixed' units. Something like this:

_So, the date picker fetches a reduced set of time slots for the calendar months if the unit type is 'hour' or 'fixed'. This means that, when a user selects one of the days with some availability on the date picker, the app needs to make another query to retrieve all the time slots available on that day._